### PR TITLE
💄Feature/15/news page

### PR DESCRIPTION
**Pull request by ashby-git**
**What did you do?**
- Created a news page, using the news-content-group component and it's props.
- But still need to add background leaf, (which sits between news and message pages) - perhaps make another component combing both news and message pages, and add the bg leaf there.

**Screenshot**
12  NewsPage

**How to Check Feature/Bug Fix**
```
import './App.css';
import NewsPage from './components/news-page/news-page.component';

function App() {
  return (
      <NewsPage />
  );
}

export default App;
```

**Link to the Design**
(https://xd.adobe.com/view/c0f0ee74-781d-434f-9960-3ff85a002849-95a3/specs/?fbclid=IwAR3bcz-BSXJGrgTXDSVBv0y-PYnSKaPM075W3UEhqyWRb_iHMHv7nu77Hq4)

### CODE REVIEW

File: src/components/news-page/news-page.component.jsx [hunk 1]
```diff
@@ -7,68 +7,77 @@ import newsImage3 from '../../assets/news_img_03.jpg';
...
-                    mediaType="Press Release"
-                />
-                <NewsContentGroup
-                    url="https://www.bbc.co.uk/news"
-                    image={newsImage6}
-                    article="新商品のお知らせ"
-                    date="2019.10.27"
-                    mediaType="Press Release"
-                />
+            <div className="container-container">
```
Comment (DanArcherOnline) [inline src/components/news-page/news-page.component.jsx:23]: try asking yourself what this thing is doing and then name the class on that.

File: src/components/news-page/news-page.component.jsx [hunk 2]
```diff
@@ -7,68 +7,77 @@ import newsImage3 from '../../assets/news_img_03.jpg';
...
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage6}
+                        article="新商品のお知らせ"
+                        date="2019.10.27"
+                        mediaType="Press Release"
+                    />
+
+                </div>
+            </div>
+            <img className="news-bg-leaf" src={messageLeaf} alt="bg-leaf" />
```
Comment (DanArcherOnline) [inline src/components/news-page/news-page.component.jsx:71]: when the browser is squished, the image seems to stop the AnimatedButton from being pressed.

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
looking good!

File: src/components/news-page/news-page.component.jsx (hunk 1 above)
Comment (ashby-git) [reply src/components/news-page/news-page.component.jsx]: whoops, that was just a temporary name. 
Changed it.

File: src/components/news-page/news-page.component.jsx (hunk 2 above)
Comment (ashby-git) [reply src/components/news-page/news-page.component.jsx:71]: fixed

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
cool cool!
one last thing; you need to change the line height of the news content group to match the design. right now the text lines are almost touching each other.

**Pull request merged**